src/components/media-player/ha-media-player-browse.ts

```diff
@@ -101,8 +101,6 @@ export class HaMediaPlayerBrowse extends LitElement {
 
   @query(".content") private _content?: HTMLDivElement;
 
-  @queryAll(".lazythumbnail") private _thumbnails?: HaCard[];
```

Comment:
Also remove the class from the elements, and remove `data-src`

Reply:
oh I guess this may still be used... for lists

Reply:
`_intersectionObserver` not used anymore?

Reply:
Or just virtualize the list too, it is easy :-)